src/test/java/com/lightstep/opencensus/exporter/LightStepExporterHandlerTest.java

```diff
@@ -82,6 +83,15 @@ public void export() throws Exception {
     LightStepTraceExporter.unregister();
   }
 
+  @Test
+  public void traceIdToLong() {
+    LightStepExporterHandler handler = new LightStepExporterHandler(null);
+
+    final TraceId traceId = TraceId.fromLowerBase16("463ac35c9f6413ad48485a3953bb6124", 0);
```

Comment:
Could you add a note on the specified value? I.e. that it's right most part actually is the specified value below.

Reply:
added